### DISCUSSION TITLE
Is it possible to use this with Expo without ejecting?

### DISCUSSION THREAD

**Discussion by RM-Eric** [invertase/react-native-firebase#8080] [General]
Is it possible to use this library with Expo without ejecting? The documentation (from what I've found) doesn't directly say if you must eject or if there is a way to use it while also staying within a managed expo project

**Comment by mikehardy** [comment]
It is possible to use this without ejecting, you should notice Expo instructions that mention the required plugins (which are hosted in this repo). I do not believe there is any need to eject anymore, and if there is, we would welcome PRs to the Expo plugins - they are community driven with PRs from people using Expo but hosted happily here
